scripts/run_group_creation.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_webapps.settings")
+django.setup()
+
+from api.algorithms.group_manager import GroupManager  # noqa: E402
+
+if __name__ == "__main__":
+    for teacher in GroupManager.get_available_teachers():
```

Comment:
Let's do it as a django management-command? https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/

Reply:
That is cool, thanks for suggestion! I was suspecting there should've been something better than manual Django init!